services/cinema_hall.py

```diff
@@ -0,0 +1,22 @@
+from django.db.models import QuerySet
```

Comment:
Add a blank line between different import groups here and in the other files to follow PEP-8 import rules